Update readme (#894)
MESMER-group/mesmer: README.rst

humans decide to follow. An interface between MESMER and global mean temperature
emulators can be found at https://github.com/MESMER-group/mesmer-openscmrunner.

> At the moment, we provide infrastrucutre to emulate annual and monthly near-surface air

Reviewer: ```suggestion
MESMER provides infrastructure to emulate annual and monthly near-surface air
```

temperature (MESMER and MESMER-M) as well as various climate extreme indicators (MESMER-X).

MESMER is under active development both scientifically and technically. Future work will
increase its user friendliness and extend its emulation capabilities to include
additional emulation methods and target climate variables.

Citing MESMER
-------------

> Scientific publications using MESMER Software should cite the following publication:

Reviewer: Typical 🐔-🥚 problem 🙂 Maybe just add the publication with title etc. but leave the doi and link out for now.

Author: yeah alright

  placeholder for MESMER publication, to be added once published.

Depending on which application of MESMER you use (MESMER, MESMER-M, MESMER-X, etc.) please
also cite the relevant publications found in our publication list on

> `readthedocs <https://mesmer-emulator.readthedocs.io/en/latest/publications.html>`_.

Reviewer: ```suggestion
also cite the relevant publications found in our
`publication list <https://mesmer-emulator.readthedocs.io/en/latest/publications.html>`_.
```
